css/styles.css

```diff
@@ -77,3 +89,180 @@ body {
   padding: 0;
   margin: 0;
 }
+
+.works {
+  padding-inline: 24px;
+}
+
+.works-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  gap: 114px;
+  margin: 114px 0;
+  justify-content: center;
+}
+
+.card {
+  min-width: calc(327px - 16px);
+  min-height: calc(552px - 16px);
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-container {
+  width: 100%;
+  height: 220px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-container img {
+  max-width: 90%;
+}
+
+.info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.info-title {
```

Comment:
Great job with font-weight, family and size 👍  But let's make sure we have the same color with the Figma value.
<img width="512" alt="Screenshot 2023-03-24 at 11 07 54 PM" src="https://user-images.githubusercontent.com/25887253/227629022-8145a208-20cf-4da5-9e62-220916353fb1.png">
<img width="512" alt="Screenshot 2023-03-24 at 11 10 25 PM" src="https://user-images.githubusercontent.com/25887253/227629060-40fdb3b5-5b7c-4679-8761-06a1281a250b.png">